.github/workflows/build_and_publish_call.yaml

```diff
@@ -0,0 +1,99 @@
+name: Build and Publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      ubuntu-version:
+        description: "Ubuntu version for the base image"
+        default: "22.04"
```

Comment:
did not we consider 20.04 to use?

Reply:
20.04 does not come with python 3.10 I would need to add ppa death snakes and install it from there .... 20.04 will install 3.8 but would have to adjust Dockerfile

Reply:
agreed

Reply:
removed